plugins/module_utils/network/eos/config/acls/acls.py

```diff
@@ -527,8 +527,6 @@ def add_commands(want):
                     )
                 if "port_protocol" in ace["source"].keys():
                     for op, val in ace["source"]["port_protocol"].items():
-                        if val.isdigit():
-                            val = socket.getservbyport(int(val))
                         command = command + " " + op + " " + val.replace("_", "-")
```

Comment:
Instead of removing the translation completely, could we keep a list of EOS implemented names as a constant? If the translated name is in the list we keep it, and if not revert to the original value.

Reply:
so it looks like we had various changes along the years so it wouldn't be a trivial change to make, however if users want to use names instead of numbers they could just use names in the yaml input right?

Reply:
upon further investigation I've found out that we have a base list of port numbers that has been there since day 1 and added all those in a list as a constant and validate if the value is a digit and part of our base list then it can do the translation otherwise it can't, tested on 4.22.4 and 4.30.1